service/controller/v2/arm_templates/vmss.json

```diff
@@ -233,11 +233,7 @@
                                                 "subnet": {
                                                     "id": "[parameters('vmssVnetSubnetId')]"
                                                 },
-                                                "loadBalancerBackendAddressPools": [
-                                                    {
-                                                        "id": "[parameters('vmssLbBackendPoolId')]"
-                                                    }
-                                                ]
+                                                "loadBalancerBackendAddressPools": "[parameters('vmssLbBackendPoolIds']"
```

Comment:
Is it tested? It should be `[parameters('vmssLbBackendPoolIds')]` instead of `[parameters('vmssLbBackendPoolIds']`.